### PR TITLE
Move errors to tilia.errors

### LINKED ISSUE #134: Refactor strings passed to display_error to tilia.errors

### PR DESCRIPTION
**Pull request by azfoo**
closes #134

### CODE REVIEW

**Review by FelipeDefensor — CHANGES_REQUESTED** [review]
This is great. I'm very relieved to see all those bare strings gone from the middle of the logic. Just see the two comments.

File: tilia/file/file_manager.py [hunk 1]
```diff
@@ -133,13 +134,10 @@ def on_import_media_metadata_request(self, path: Path | str) -> None:
             with open(path, "r", encoding="utf-8") as f:
                 data = json.load(f)
         except json.decoder.JSONDecodeError as err:
-            post(
-                Post.DISPLAY_ERROR,
-                f"Error when parsing file {path}:\n{err}",
-            )
+            post(Post.DISPLAY_ERROR, *tilia.errors.MEDIA_METADATA_IMPORT_JSON_FAILED, path, repr(err))
```
[the remark this answers is not in the document]
Comment (FelipeDefensor) [reply tilia/file/file_manager.py]: That's poor test desing on my part. I fixed that on 79ee289.

**Comment by FelipeDefensor** [comment]
I noticed there are some posts to fix at these modules:
media/player/base.py
ui/timelines/collection/requests/args.py
ui/cli/metadata/set_media_length.py

There's a last failing test that should be fixed by that.

**Comment by azfoo** [comment]
Could I possibly ignore the last error in player for now? It is fixed in the other branch.

**Comment by FelipeDefensor** [comment]
> Could I possibly ignore the last error in player for now? It is fixed in the other branch.

If that's the case, sure.

**Review by FelipeDefensor — APPROVED** [review]
All good. Just make sure to squash before merging.

**Pull request merged**